frontend/public/components/cron-job.jsx

```diff
@@ -45,27 +45,23 @@ const Details = ({obj: cronjob}) => {
           <dt>Schedule</dt>
           <dd>{cronjob.spec.schedule}</dd>
           <dt>Concurrency Policy</dt>
-          <dd>{_.get(cronjob.spec, 'concurrencyPolicy', '-')}</dd>
+          <dd>{cronjob.spec.concurrencyPolicy || '-'}</dd>
           <dt>Starting Deadline Seconds</dt>
-          <dd>{_.get(cronjob.spec, 'startingDeadlineSeconds', '-')}</dd>
+          <dd>{cronjob.spec.startingDeadlineSeconds || '-'}</dd>
+          <dt>Last Schedule Time</dt>
+          <dd><Timestamp timestamp={cronjob.status.lastScheduleTime} /></dd>
         </ResourceSummary>
       </div>
       <div className="col-md-6">
         <SectionHeading text="Job Overview" />
-        <ResourceSummary resource={cronjob} showNodeSelector={false}>
+        <dl className="co-m-pane__details">
           <dt>Desired Completions</dt>
           <dd>{job.spec.completions || '-'}</dd>
           <dt>Parallelism</dt>
           <dd>{job.spec.parallelism || '-'}</dd>
           <dt>Deadline</dt>
           <dd>{job.spec.activeDeadlineSeconds ? `${job.spec.activeDeadlineSeconds} seconds` : '-'}</dd>
-          <dt>Status</dt>
```

Comment:
I think that it also would make sense to show  `restartPolicy`, mainly cause this field is always present...

Reply:
That is part of the pod template, and we're not showing any info about the containers currently.

Reply:
I opened CONSOLE-736 to add container info

Reply:
Yeah it's a great point, and I think generally the workloads pages need more detail. We should be careful how we do it though because pod templates have a lot of fields that can clutter the UI, so I didn't want to do too much in this PR.

Thanks @jhadvig